src/main/java/de/hs_mannheim/informatik/ct/controller/ContactTracingController.java

```diff
@@ -64,14 +64,73 @@
+            new TracingColumn("Mtknr.", contact -> contact.getContact().getEmail().split("@")[0]),
             new TracingColumn("Raum/Veranstaltung", Contact::getContactLocation),
             new TracingColumn("Datum", contact -> dateFormatter.format(contact.getTargetVisit().getStartDate())),
             new TracingColumn("Anmeldung Ziel", contact -> timeFormatter.format(contact.getTargetVisit().getStartDate())),
             new TracingColumn("Anmeldung Kontakt", contact -> timeFormatter.format(contact.getContactVisit().getStartDate())),
             new TracingColumn("Abmeldung Ziel", contact -> timeFormatter.format(contact.getTargetVisit().getEndDate())),
-            new TracingColumn("Abmeldung Kontakt", contact -> timeFormatter.format(contact.getContactVisit().getEndDate()))
+            new TracingColumn("Abmeldung Kontakt", contact -> timeFormatter.format(contact.getContactVisit().getEndDate())),
+            new TracingColumn("Geburtsdatum", contact -> ""),
+            new TracingColumn("Straße", contact -> ""),
+            new TracingColumn("Hausnummer", contact -> ""),
+            new TracingColumn("Plz", contact -> ""),
+            new TracingColumn("Ort", contact -> ""),
+            new TracingColumn("Tel.", contact -> "")
+    );
+    private final List<TracingColumn> tracingColumnsStaff = Arrays.asList(
+            new TracingColumn("EMail-Adresse", contact -> contact.getContact().getEmail()),
+            new TracingColumn("Name", contact -> contact.getContact().getEmail().split("@")[0]),
+            new TracingColumn("Raum/Veranstaltung", Contact::getContactLocation),
+            new TracingColumn("Datum", contact -> dateFormatter.format(contact.getTargetVisit().getStartDate())),
+            new TracingColumn("Anmeldung Ziel", contact -> timeFormatter.format(contact.getTargetVisit().getStartDate())),
+            new TracingColumn("Anmeldung Kontakt", contact -> timeFormatter.format(contact.getContactVisit().getStartDate())),
+            new TracingColumn("Abmeldung Ziel", contact -> timeFormatter.format(contact.getTargetVisit().getEndDate())),
+            new TracingColumn("Abmeldung Kontakt", contact -> timeFormatter.format(contact.getContactVisit().getEndDate())),
+            new TracingColumn("Geburtsdatum", contact -> ""),
+            new TracingColumn("Straße", contact -> ""),
+            new TracingColumn("Hausnummer", contact -> ""),
+            new TracingColumn("Plz", contact -> ""),
+            new TracingColumn("Ort", contact -> ""),
+            new TracingColumn("Tel.", contact -> "")
+    );
+    private final List<TracingColumn> tracingColumnsGuests = Arrays.asList(
+            new TracingColumn("EMail-Adresse", contact -> contact.getContact().getEmail()),
+            new TracingColumn("Nachname", contact -> contact.getContact().getName().split(" ")[0]),
+            new TracingColumn("Vorname", contact -> contact.getContact().getName().split(" ")[1]),
```

Comment:
I think, the problem lies in the assumption that there is always a first an a last name entered, which is not enforced in any way. Hence, I would leave this in one column together.

Moreover, I just changed the order to Vorname Nachname in the input field today as I find it strange to do it the other way round.